packages/components/src/sender/extensions/template/block/plugins.ts

```
@@ -0,0 +1,651 @@
+          if (currentNode.type.name === 'template') {
+            const content = currentNode.textContent || ''
+
+            // 删除第一个字符时，插入零宽字符（保留模板块）
+            if (
+              $from.pos === $from.start() &&
+              content.length === 1 &&
+              content !== ZERO_WIDTH_CHAR &&
+              event.key === 'Delete'
+            ) {
+              const pos = $from.pos
+              dispatch(state.tr.insertText(ZERO_WIDTH_CHAR, pos, pos + 1))
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)